testing_exercise/testing.js

```diff
@@ -0,0 +1,41 @@
+function replaceWith(str, target, replacement){
+	return str
+		.split('')
+		.map(function(v,i,a){
```

Comment:
small thing, but if you're not using the other arguments `i` and `a`, you don't need to include them in the callback. your callback can just be a function of a single parameter.